### DISCUSSION TITLE
Jenkins Distributed Task Execution (DTE) from SDK

### DISCUSSION THREAD

**Discussion by ild0tt0re** [dagger/dagger#12517] [Ideas]
From a "CI/CD Pipelines as Code" system, I expect to dinamically handle jenkins agents from code. 😍

Example:
Run build, test, lint, typecheck commands on 4 or more agents based on how many subtasks have each target (i.e: if my unit test without cache take 20 min, I would like to split them on furtherer two different agent)

**Comment by mircubed** [comment]
related to #4561
